### PR TITLE
Load pow library internally instead calling subprocess

### PR DESCRIPTION
**Pull request by chenwei-tw**

### CODE REVIEW

**Comment by jserv** [comment]
@chenwei-tw, can you rebase to latest Git master branch?

**Comment by chenwei-tw** [comment]
Done. Could @yillkid  give it a check?

File: swarm_node.py [hunk 1]
```diff
@@ -41,6 +41,10 @@ def get_tips(tips_type):
 	return api.get_tips()
 
 def send_transfer(tag, messages, address, values, dict_tips, debug=0):
+    ## Initialize PoW Library
+    PoWlib = PoW_load_library("./libdcurl.so")
```
Comment (jserv) [inline swarm_node.py]: Should be `deps/dcurl/build/libdcurl.so`

**Comment by chenwei-tw** [comment]
Done.

**Pull request merged**